x-pack/plugins/endpoint/server/routes/metadata/query_builders.test.ts

```diff
@@ -3,12 +3,13 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
+
```

Comment:
Moved this file. The 'service' module is expected to contain New Platform 'services', which are values shared at runtime w/ other plugins.